client/api.py

```diff
@@ -25,14 +25,14 @@ def health_check():
 
 @app.route('/website', methods=['GET'])
 def get_metadata():
-    query_params = QueryParams()
+    urls = request.args.getlist('src')
+    query_params = QueryParams(urls)
     local_request = request
 
-    url = request.args.get('src').strip()
     response_type = local_request.args.get('format')
     desc_length = local_request.args.get('desc_cap')
 
-    query_params.query_url = url
+    query_params.query_url = urls
```

Comment:
I don't get, what do you mean one variable? I only changed it from url to urls, coz it can be multiple. @MeenuV